server.js

```diff
@@ -1,6 +1,13 @@
-//require("dotenv").config();
+require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
+//const morgan = require('morgan');
+const session = require("express-session");
+const passport = require("passport");
+const indexRoutes = require('./routes/index');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
const indexRoutes = require("./routes/index");
```